cmu_graphics/cmu_graphics.py

```diff
@@ -590,6 +590,7 @@ def onResize(self, newScreen=True):
             return
         self.updateScreen(newScreen)
         self.callUserFn('onResize', ())
+        self.redrawAllWrapper()
```

Comment:
I think a nice solution here is to update the `onResize` function to this:

```python3
def onResize(self, newScreen=True):
    if not self._running:
        return
    self.updateScreen(newScreen)
    self.callUserFn('onResize', (), redraw=False)
    if self._isMvc:
        self.redrawAllWrapper()
```

`callUserFn` usually causes the screen to redraw, but when a user function isn't defined, it returns early. So this code actually worked correctly!

```python3
from cmu_graphics import*

def redrawAll(app):
    drawRect(app.width/2-app.width/10,app.height/2-app.height/10,app.width/5,app.height/5)

def onResize(app):
    pass

runApp()
```

Passing `redraw=False` to `callUserFn` guarantees that it won't draw, so we don't risk drawing twice in some cases. And only calling `redrawAllWrapper` when we're in "MVC Mode" means that we won't accidentally clear the screen for CS1 graphics users.

@Mohamed-Waiel-Shikfa if you make those edits to this branch, I think we can merge this PR. If not, I'll open a PR of my own later this week :)

Reply:
Hey!
I now understand better the reason behind deleting it in the first place. Thanks.
I have made the change and committed it, so it should be ready to be pulled. 
If possible, it would be great to make a new release after that, so students can work on their projects properly.

Off-topic, but I would like to contribute more towards this software. If you have any guidelines or procedures to get started, it would be great. I am willing to work on anything, be it bug fixing, implementing additional features, etc.
Let me know by email 😁 (my email is in my profile).